spec/integration/monorepo/after/gen/A/Pods.xcodeproj

```diff
@@ -356,16 +348,14 @@ Targets:
         Base Configuration: A-iOS.app.xcconfig
         Build Settings:
           ASSETCATALOG_COMPILER_APPICON_NAME: AppIcon
+          CLANG_ENABLE_OBJC_WEAK: NO
           CODE_SIGN_IDENTITY: iPhone Developer
-          "CODE_SIGN_IDENTITY[sdk=appletvos*]": ''
-          "CODE_SIGN_IDENTITY[sdk=iphoneos*]": ''
-          "CODE_SIGN_IDENTITY[sdk=watchos*]": ''
           CODE_SIGNING_ALLOWED: YES
           CODE_SIGNING_REQUIRED: YES
           CURRENT_PROJECT_VERSION: '1'
           GCC_PREFIX_HEADER: Target Support Files/A-iOS/A-iOS-App-prefix.pch
           INFOPLIST_FILE: App/A-iOS-App-Info.plist
-          IPHONEOS_DEPLOYMENT_TARGET: '8.0'
```

Comment:
see below https://github.com/square/cocoapods-generate/pull/39/files#r322496531 why this is happening and was fixed.

Reply:
fixed.